packages/dashboard-frontend/src/Layout/ErrorReporter/Issue/index.tsx

```diff
@@ -92,6 +97,55 @@ export class IssueComponent extends React.PureComponent<Props> {
     );
   }
 
+  private renderWorkspaceStoppedError(
+    error: Error,
+    workspaceRoutes: WorkspaceRoutes | undefined,
+  ): React.ReactNode {
+    const linkOnClick = (hash: string) => {
+      return () => {
+        window.location.hash = hash;
+        window.location.reload();
+      };
+    };
+
+    let ideLoader;
+    let workspaceDetails;
```

Comment:
Could you declare variables with types?

Reply:
Sounds great, my push sets the types to `React.ReactNode`